### ISSUE TITLE
Combine all requirements into one doc from all notes

### ISSUE THREAD

**Issue by AyushKharade** [ArchambaultP/Comp354Website#13]
Combine notes from everyone who took it during the interview and sort it according to core / optional and importance. Doc files are on slack

**Comment by ArchambaultP** [comment]
Yeah, I'll aggregate all the doc notes together. However, there will be a design document detailing everything that needs to be done tech-wise. That way, you guys don't have to get bogged-down with reading all the specs/
